src/api/solver-network/content-types/solver-network/schema.json

```diff
@@ -41,6 +41,12 @@
       "type": "relation",
       "relation": "oneToOne",
       "target": "api::environment.environment"
+    },
+    "isWhiteListed": {
+      "type": "boolean"
+    },
+    "isVouched": {
+      "type": "boolean"
```

Comment:
same for these 2, I would make it mandatory as we don't want to model the "unknown" (either they are, or not)